nx.json

```diff
@@ -0,0 +1,43 @@
+{
+    "extends": "nx/presets/npm.json",
+    "tasksRunnerOptions": {
+        "default": {
+            "runner": "@nrwl/nx-cloud",
+            "options": {
+                "cacheableOperations": [
+                    "lint",
+                    "test:unit",
+                    "type-check",
+                    "lint:js",
+                    "lint:styles",
+                    "build:lib"
+                ],
+                "accessToken": "MzdkNzIwNjUtMWUyZC00YTU2LThlY2YtYjQzYzU0ZThjNTg0fHJlYWQtd3JpdGU=",
```

Comment:
Maybe there is a way not to share access token publicly? This way we need to be very sure, that we don't rely on it for release builds.

Reply:
I think it doesn't matter for this kind of token, it's storing just hashes of packages, not actual build outputs.

Reply:
I will generate read only token that will be here and generate new write token that will be set using env variables.

Reply:
Old token was invalidated, new one is read-only and second new one with RW rights will be distributed using env variable.